Update dependency ets-gml32 to latest version

## Test with SNAPSHOT version of current ets-gml32 master (includes support of GML 3.2.2)

Used master branch of ets-wfs20 for test.

Dependency was updated in pom file:
```
stenger@elsbeere:~/git/ets-wfs20$ git diff
diff --git a/pom.xml b/pom.xml
index b4ce4a1..1e8c940 100644
--- a/pom.xml
+++ b/pom.xml
@@ -66,7 +66,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>ets-gml32</artifactId>
-      <version>1.29</version>
+      <version>1.31-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.opengis.cite.teamengine</groupId>
```

Afterwards, test suite was started via Docker.

When testing against reference implementation [1], several tests fail:
* compile XML Schema | java.lang.IllegalArgumentException
* property Is Not Like | java.lang.AssertionError: Test cannot be executed as no schema can be found; Please check if DescribeFeatureType returns a valid schema.
  * This error message occurs multiple times.

Thus, the update of ets-gml32 leads to some unexpected problems.
@bpross-52n Can you please take a look at those problems?

[1] https://cite.deegree.org/deegree-webservices-3.4.30/services/wfs200?service=WFS&request=GetCapabilities

--- a/pom.xml
+++ b/pom.xml
@@ -66,7 +66,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>ets-gml32</artifactId>
-      <version>1.29</version>
+      <version>1.31</version>
     </dependency>
     <dependency>
       <groupId>org.opengis.cite.teamengine</groupId>

